### DISCUSSION TITLE
What's the recommended way to handle cookie-based authentication?

### DISCUSSION THREAD

**Discussion by ulisesvina** [tauri-apps/tauri#10214] [Q&A]
I have a REST API that has Lucia Auth implemented, so on successful login it returns an http-only cookie with the token, so what's the recommended way to handle this in Tauri? I mean, changing the state to logged or unauthenticated, saving the user's data, and other auth-based data storing.

**Comment by sam-brainstream** [comment]
Don't use cookies for authentication in Tauri. It only works in dev mode.
